deployments/azure-kubernetes-service-helm.yml

```diff
@@ -9,16 +9,17 @@
 # To configure this workflow:
 #
 # 1. Set the following secrets in your repository (instructions for getting these 
-#    https://github.com/Azure/login#configure-a-service-principal-with-a-federated-credential-to-use-oidc-based-authentication):
+#    https://docs.microsoft.com/en-us/azure/developer/github/connect-from-azure?tabs=azure-cli%2Clinux)):
 #    - AZURE_CLIENT_ID
 #    - AZURE_TENANT_ID
 #    - AZURE_SUBSCRIPTION_ID
 #
 # 2. Set the following environment variables (or replace the values below):
 #    - AZURE_CONTAINER_REGISTRY (name of your container registry / ACR)
+#    - CONTAINER_NAME (name of the container image you would like to push up to your ACR)
 #    - RESOURCE_GROUP (where your cluster is deployed)
 #    - CLUSTER_NAME (name of your AKS cluster)
-#    - CONTAINER_NAME (name of the container image you would like to push up to your ACR)
```

Comment:
This undoes @imiller31's previous [PR](https://github.com/actions/starter-workflows/pull/1494) which removed image pull secrets. Is that really intended?

Reply:
Also I noticed that this PR and the previous PR both are targeting `partner_templates` branch which is no longer in use. Please use `main` branch for any PR. Since we also don't have

Reply:
We are undoing Israel's changes because attaching the ACR only works with [owner permissions](https://github.com/azure/aks/issues/1517). We discussed together and decided this was the best path forward until that gets fixed.

Reply:
@bishal-pdMSFT [link to the new PR](https://github.com/actions/starter-workflows/pull/1534)